src/main/java/org/codehaus/groovy/runtime/StringGroovyMethods.java

```diff
@@ -3888,4 +3888,385 @@ public static boolean isBlank(CharSequence self) {
 
         return self.toString().matches("\\s*");
     }
+
+	/**
+	 * Returns the last <code>num</code> elements from this CharSequence.
+	 * <pre class="groovyTestCase">
+	 * def text = "Groovy"
+	 * assert text.takeRight( 0 ) == ''
+	 * assert text.takeRight( 2 ) == 'vy'
+	 * assert text.takeRight( 7 ) == 'Groovy'
+	 * </pre>
+	 *
+	 * @param self the original CharSequence
+	 * @param num  the number of chars to take from this CharSequence from the right
+	 * @return a CharSequence consisting of the last <code>num</code> chars,
+	 *         or else the whole CharSequence if it has less than <code>num</code> elements.
+	 */
+	public static CharSequence takeRight(CharSequence self, int num) {
+    	if (num < 0)
+    		return self.subSequence(0, 0);
+
+	    int begin = Math.max(0, self.length() - num);
+	    return self.subSequence(begin, self.length());
+    }
+
+	/**
+	 * A GString variant of the equivalent CharSequence method {@link #takeRight(GString, int)}
+	 *
+	 * @param self the original GString
+	 * @param num  the number of chars to take from this GString from the right
+	 * @return a String consisting of the last <code>num</code> chars,
+	 *         or else the whole GString if it has less than <code>num</code> elements.
+	 */
+	public static String takeRight(GString self, int num) {
+		return (String) takeRight(self.toString(), num);
+	}
+
+	/**
+	 * Returns the {@code CharSequence} that exists after the first occurrence of the given
+	 * {@code searchString} in this CharSequence
+	 *
+	 * <pre class="groovyTestCase">
+	 * def text = "Groovy development. Groovy team"
+	 * assert text.after( 'Groovy' )           == ' development. Groovy team'
```

Comment:
these should fail if you run the full test suite since they don't reflect the name change

Reply:
This line is from un-corrected javadoc. corrected now
```suggestion
	 * assert text.takeAfter( 'Groovy' )           == ' development. Groovy team'
```